pkg/adapter/nimbus-kyverno/utils/utils.go

```diff
@@ -4,16 +4,22 @@
 package utils
 
 import (
+	"encoding/json"
 	"fmt"
+	"os"
 	"reflect"
 	"strings"
+	"sync"
 
 	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var VirtualPatchData []map[string]any
```

Comment:
Don't you think we should define a type for type safety instead of `map[string]interface{}`?

Reply:
We don't know whether the data will be of map[string]map[string]string or map[string]string or map[string][int]map[string]string